azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -321,6 +372,7 @@ func resourceArmKubernetesClusterCreate(d *schema.ResourceData, meta interface{}
 	agentProfiles := expandAzureRmKubernetesClusterAgentProfiles(d)
 	servicePrincipalProfile := expandAzureRmKubernetesClusterServicePrincipal(d)
 	networkProfile := expandAzureRmKubernetesClusterNetworkProfile(d)
+	addOnProfile := expandAzureRmKubernetesClusterAddOnProfile(d)
```

Comment:
Am I being nit-picky to request this to be addonProfiles instead of addOnProfile? (just to reflect AddonProfiles with camel-casing)

Reply:
not at all - the schema field is `addon_profile` to match the general convention, but I can push a commit to update this :)

Reply:
Thanks Tom! :smile:

Reply:
pushed: 
https://github.com/terraform-providers/terraform-provider-azurerm/pull/1751/commits/be893dc2d997fdca3abe8b4e3106b23dbe973e98 :)